pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -76,14 +78,15 @@ func (r *Reconciler) ValidateMachineConfigs(ctx context.Context, log logr.Logger
 	log = log.WithValues("phase", "validateMachineConfigs")
 	for _, machineConfig := range clusterSpec.SnowMachineConfigs {
 		if !machineConfig.Status.SpecValid {
-			failureMessage := fmt.Sprintf("SnowMachineConfig %s is invalid", machineConfig.Name)
 			if machineConfig.Status.FailureMessage != nil {
```

Comment:
combine with the prev `if` statement?

Reply:
I think that wouldn't have the effect you expect it to have. If I combine those, I can't handle the case where `SpecValid` is false but the failure message is nil. In that case we also want to stop the reconciliation.

Reply:
Ah yes I see. Ok disregard then!